src/components/map/MapService.js

```diff
@@ -362,6 +362,7 @@ goog.require('ga_urlutils_service');
           transparent: 'true'
         };
         return new Cesium.UrlTemplateImageryProvider({
+          minimumRetrievingLevel: window.minimumRetrievingLevel,
```

Comment:
Do you prefer I inject the `gaPermalink` in MapService.js and read the value from there?